dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -210,6 +213,7 @@ def build_try_again_response(options = {})
 
     params = @milestone_params
     params[:script_level_id] = @script_level.id
+    params[:course_id] = @script.original_unit_group_id
```

Comment:
necessary because the milestone route calls track_level_progress, which is now hitting this check: https://github.com/code-dot-org/code-dot-org/blob/aa85c725445ded53ac047a3ac8342ebf8d0c9a42/dashboard/app/models/user_script.rb#L82-L85